Library/Formula/librsvg.rb

```diff
@@ -42,4 +42,17 @@ def post_install
     ENV["GDK_PIXBUF_MODULEDIR"] = "#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
     system "#{Formula["gdk-pixbuf"].opt_bin}/gdk-pixbuf-query-loaders", "--update-cache"
   end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <librsvg/rsvg.h>
+
+      int main(int argc, char *argv[]) {
+        RsvgHandle *handle = rsvg_handle_new();
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-I#{HOMEBREW_PREFIX}/include/librsvg-2.0", "-I#{HOMEBREW_PREFIX}/include/gdk-pixbuf-2.0", "-I#{HOMEBREW_PREFIX}/include/libpng16", "-I#{HOMEBREW_PREFIX}/include/cairo", "-I#{HOMEBREW_PREFIX}/include/glib-2.0", "-I#{HOMEBREW_PREFIX}/lib/glib-2.0/include", "-I#{HOMEBREW_PREFIX}/opt/gettext/include", "-I#{HOMEBREW_PREFIX}/include/pixman-1", "-I#{HOMEBREW_PREFIX}/include", "-I#{HOMEBREW_PREFIX}/include/freetype2", "-I#{HOMEBREW_PREFIX}/include/libpng16", "test.c", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib", "-L#{HOMEBREW_PREFIX}/lib", "-lrsvg-2", "-lm", "-lgio-2.0", "-lgdk_pixbuf-2.0", "-lgobject-2.0", "-lglib-2.0", "-lintl", "-lcairo", "-o", "test"
```

Comment:
Please use `#{include}`, `#{lib}`, `#{Formula["foo"].opt_include}` and `#{Formula["foo"].opt_lib}` instead. And remove all unnecessary and redundant arguments. Thanks.